### PR TITLE
Add Unit Tests

### PR DESCRIPTION
**Pull request by Tahirih-Natang**
### Description
Added u-tests for forms and views in the malaria_web app. Also added u-test for forms in the pcsa_GHN web app.

### CODE REVIEW

File: malaria_web/tests.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        form = PostForm(data={'title_post': "post", 'description_post': "Description", 'link_post': "url", 'photo': "image"})
+        self.assertTrue(form.is_valid())
+
+    # Invalid Form Data
+    def test_PostForm_invalid(self):
+        form = PostForm(data={'title_post': "", 'description_post': "mp", 'link_post': "mp", 'photo': ""})
+        self.assertFalse(form.is_valid())
+
+class View_Test(TestCase):
+    #This comment is just to replace trailing whitespace
```
Comment (medhach) [inline malaria_web/tests.py]: Can you please remove this?

**Comment by medhach** [comment]
@Tahirih-Natang Tests look good for a start. Can you set these tests up to run with every travis build?
Let me know if you need help with it.

File: malaria_web/tests.py (hunk 1 above)
Comment (Tahirih-Natang) [reply malaria_web/tests.py]: Done!

**Comment by Tahirih-Natang** [comment]
All changes have been made as requested. Concerning seting the test up to run with every travis build, I will need some help with that

**Pull request merged**